### ISSUE TITLE
 tslint:disable-next-line should ignore subsequent comments

### ISSUE THREAD

**Issue by mertdeg2** [palantir/tslint#3713]
#### TypeScript code being linted
Basically, I wanna do this:
```ts
// tslint:disable-next-line:no-any very very long explanation
// on why my code is hacky.
(foo as any).bar = 5;
```

#### Actual behavior
This throws lint warning

#### Expected behavior
This does not throw a lint warning. I could do something like this:

```ts
// tslint:disable:no-any very very long explanation
// on why my code is hacky.
(foo as any).bar = 5;
// tslint:enable:no-any
```

But this results in more boilerplate for what i want to do. 

From here, it looks like this would be easy to change: https://github.com/palantir/tslint/blob/235561e411c58ecbd4d96844fae28e998f0ccbda/src/enableDisableRules.ts#L113-L121, But I am not sure.

**Comment by JoshuaKGoldberg** [comment]
☠️ *TSLint's time has come!* ☠️ 

TSLint is no longer accepting most feature requests per #4534. See **[typescript-eslint.io](https://typescript-eslint.io)** for the new, shiny way to lint your TypeScript code with **ESLint**. ✨ 

It was a pleasure open sourcing with you all!

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈 _(#4534)_  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
